quickget

```diff
@@ -1218,6 +1218,7 @@ function web_get() {
     if [[ ${OS} != windows && ${OS} != macos && ${OS} != windows-server ]]; then
         echo "Downloading $(pretty_name "${OS}") ${RELEASE} ${EDITION}"
         echo "- URL: ${URL}"
+        echo "- PATH: ${DIR}/${FILE}"
```

Comment:
```suggestion
        echo "- PATH: ${PWD}/${DIR}/${FILE}"
```
Maybe useful for those with several collections of VMs dotted around the filesystem or on several external drives